tests/test_cadquery.py

```diff
@@ -5790,3 +5791,41 @@ def test_tessellate(self):
         verts, _ = Face.makePlane(1e-9, 1e-9).tessellate(1e-3)
 
         assert len(verts) == 0
+
```

Comment:
Not a must, but the `rarray` call on lines 5760, 5768, prompted me to open issue #1603.  These *could* be updated to set y spacing to zero for readability.